packages/backend/src/openapi/specs/webhooks.yaml

```diff
@@ -341,3 +352,29 @@ components:
         scale:
           type: number
       additionalProperties: false
+    amount:
+      title: amount
+      type: object
+      properties:
+        value:
+          type: string
+          format: uint64
+          description: 'The value is an unsigned 64-bit integer amount, represented as a string.'
+        assetCode:
+          $ref: '#/components/schemas/assetCode'
+        assetScale:
+          $ref: '#/components/schemas/assetScale'
+      required:
+        - value
+        - assetCode
+        - assetScale
+    assetCode:
+      title: Asset code
+      type: string
+      description: The assetCode is a code that indicates the underlying asset. This SHOULD be an ISO4217 currency code.
+    assetScale:
+      title: Asset scale
+      type: integer
+      minimum: 0
+      maximum: 255
+      description: The scale of amounts denoted in the corresponding asset code.
```

Comment:
we can move these directly inside the amount schema (ie these don't need to be in their separate schema)